spring-boot-starter/spring-boot-starter-discovery/src/main/java/org/apache/servicecomb/springboot/starter/discovery/CseDiscoveryClient.java

```diff
@@ -44,24 +51,55 @@ public String description() {
 
   @Override
   public List<ServiceInstance> getInstances(final String serviceId) {
+    class InstanceDiscoveryFilter implements DiscoveryFilter {
```

Comment:
put class InstanceDiscoveryFilter  in "getInstances", make "getInstances" logic not clear

Reply:
Will move to a separate class